tf2onnx/tfonnx.py

```diff
@@ -216,6 +216,12 @@ def rewrite_dropout(g, ops):
         for n in set(match.get_nodes()):
             g.remove_node(n.name)
 
+    # remove dropout if its ratio is 1.0
+    for node in g.get_nodes():
```

Comment:
no need to go through all nodes, we can get all dropout node names easily.

Reply:
don't want to mix these two (actually) rewriter up.

Reply:
make sense.